plugins/memcached/memcachedplugin.cpp

```diff
@@ -640,6 +650,11 @@ ECL_MEMCACHED_API const char * ECL_MEMCACHED_CALL MKeyType(ICodeContext * ctx, c
     const char * keyType = enumToStr(serverPool->getKeyType(key, partitionKey));
     return keyType;
 }
+ECL_MEMCACHED_API void ECL_MEMCACHED_CALL MDelete(ICodeContext * ctx, const char * key, const char * options, const char * partitionKey)
```

Comment:
Observation: Too late to make a change, but just looking at the calls - the memcached interface is assuming a (length, string) argument for its calls, so passing parameters as const string instead of const varstring would probably have been a better choice.  Not critical enough to be worth worrying about.